tests/baselines/reference/enumErrors.js

```diff
@@ -86,10 +88,11 @@ var E10;
 // Enum with computed member intializer of other types
 var E11;
 (function (E11) {
-    E11[E11["A"] = true] = "A";
-    E11[E11["B"] = new Date()] = "B";
-    E11[E11["C"] = window] = "C";
-    E11[E11["D"] = {}] = "D";

```

Comment:
I think these are probably an issue, and likely due to that `return` - maybe you could `if`/`else` instead to avoid duplicate errors?

Reply:
I fixed it! Thank you for your comment!
https://github.com/microsoft/TypeScript/pull/37790/commits/c2d15a8978c70aec488429c31e83ef61f53d0a84